help/C/Help_ch_Customize.xml

```diff
@@ -924,7 +924,7 @@
           <itemizedlist>
 
             <listitem>
-              <para><guilabel>Use system theme colors:</guilabel> If selected,
+              <para><guilabel>Don't use <application>&app;</application> built-in colors:</guilabel> If selected,
```

Comment:
/home/frank/git/gnucash-docs/help/C/Help_ch_Customize.xml:927: element guilabel: validity error : Element application is not declared in guilabel list of possible children
<guilabel>Don't use <application>&app;</application> built-in colors:</guilabel>
                                                                               ^
I will remove `<application>/</application>`.

@jralls, @gjanssens : Do we use travis here? If yes, does it run xmllint?

Another note: Somewhere in the docs referenced in https://wiki.gnucash.org/wiki/Translation#Further_Reading they write you should avoid "Don't" (slang) and use "Do not" instead. Should we apply this here?

Reply:
Regarding the messed up commit and branch: I am a total klutz WRT Git. I saw the additional commits, but I do not know how that happened, and did not/do not know how to do it differently. Sorry about that.

Regarding embedding the application symbol: that too is my own mistake. Sorry.

As for use of a possessive; I don't think it's that important. I simply transcribed what was there. And, the use of "Don't" is also not something I feel strongly about. As before, I simply transcribed what was there.

Reply:
@fellen The docs are not checked on Travis. It shouldn't be too difficult to add.

Reply:
I'm fine with changing "Don't" to "Do not". However I would even more prefer to find a suitable option name that's not using a negation, while not reverting the default meaning of the option.

That is, the ideal wording would have been
"Use gnucash built-in colors"

But if the existing option was set to true (to effectively not use gnucash built-in color scheme), this true value would suddenly mean the opposite if we changed the wording to "Use gnucash built-in colors".

So such a name can't be done considering existing users have set this option to their preference already (either implicitly or explicitly).

The [original bug report](https://bugzilla.gnome.org/show_bug.cgi?id=746163) that triggered this change suggested another wording still (albeit after the bug was already closed):
"Override GnuCash built-in colors"
While affirmative it conveys things backwards IMO. It's gnucash overriding the system provided color mechanics, not the other way around.

I'm also pondering these alternative wordings:
"Skip GnuCash built-in colors"
"Ignore GnuCash built-in colors"

Oh and "color theme" feels more precise than "colors".

I would love to hear an even better one though.

Reply:
...or "Ignore gnucash built-in colors"?
Perhaps this is a question for gnucash-user for better sugestions.

Reply:
To be honest, I think negative preferences are exceedingly grotesque. "Check this box in order to NOT do something" is just plain confusing. Having read the bug report, perhaps it would be better to have this preference read "Allow system or custom colors." Then, the user is checking a box to achieve a particular goal, rather than to avoid the opposite.

Reply:
@sunfish62 I fully agree on the negative preference issue. So I have decided to rename the option to "Use GnuCash built-in color  theme" (which reverses the meaning of the option) and handle the new meaning properly in code. I have also changed the accompanying help documentation in the same sense. This is something I could not do in the course of the 2.6 series, but can be done in a jump to a new major release.